src/App.js

```diff
@@ -13,30 +12,71 @@ class App extends React.Component {
       movies: [],
       selectedMovie: {},
       isSelected: false,
+      error: '',
+      errorMsg: ''
```

Comment:
Went down one path with error handling. Aren't currently using this but might circle back.